app/src/main/java/com/github/sdp/tarjetakuna/database/DBMagicCard.kt

```diff
@@ -10,22 +10,12 @@ import com.google.gson.Gson
 @Entity(tableName = "magic_card", primaryKeys = ["code", "number"])
 data class DBMagicCard(
     val card: String,
-    val possession: CardPossession,

```

Comment:
Removing this field will (and have) break the schema of the local database, so this is one of the reasons why the app just crashes.
But I can try to fix this by coming on your branch